.ci.yaml

```diff
@@ -1356,6 +1356,16 @@ targets:
       task_name: color_filter_cache_perf__e2e_summary
     scheduler: luci
 
+  - name: Linux_android raster_cache_use_memory_perf__e2e_summary
```

Comment:
Could you start the new task/target with `bringup: true`? Any new task should be validated in the staging pool first before enabled in prod.